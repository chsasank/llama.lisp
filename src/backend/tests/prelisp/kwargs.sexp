(define ((main int))
    (call printf ,(release_info_string "v0.1" (#:compiler "Llama.lisp/LLVM") (#:date "2024/12/23")))
    (call printf ,(release_info_string "v0.1" (#:date "2024/12/23"))))
