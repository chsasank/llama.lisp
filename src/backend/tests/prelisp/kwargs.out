(define ((main int))
  (call printf
        "Llama.lisp Test Suite | Version: v0.1 | Compiler: Llama.lisp/LLVM | Date: 2024/12/23")
  (call printf
        "Llama.lisp Test Suite | Version: v0.1 | Date: 2024/12/23"))
